File: src/kotlintest/Stdout.java
package kotlintest;

import java.util.Scanner;

class Stdout{

    public static void main(String args[]) {
        Scanner sc =new Scanner(System.in);
        int a=sc.nextInt();
        String st = sc.next();
        int len =st.length();
        if(a>=0 && a<=10)
        System.out.println(a*2);
        if(len>=1 && len<=15)
        System.out.println(st);
    }

}

File: src/hackerranktest/Stdout.kt
package hackerranktest

import java.util.*

internal object Stdout {
    @JvmStatic
    fun main(args: Array<String>) {
        val sc = Scanner(System.`in`)
        val a = sc.nextInt()
        val st = sc.next()
        if (a in 0..10) println(a * 2)
        if (st.length in 1..15) println(st)
    }
}